### PR TITLE
homepage: platform section

### LINKED ISSUE #795: New Home Page
Hi team, I am creating this issue a little bit in advance because the page is not completely finished yet. It will be this Friday or Monday. My plan is to leave the file tidy and with notes as I did with Github page to guide you.

The hero animation will be sent separately as a video to embed (we can talk about this later) 
The platform animation will be sent separately as a reference for you to imitate with scrolling animation.

https://www.figma.com/file/fTnEEueuxlkKrubDegp9aw/HP?node-id=517%3A27561

Thanks!

### PR DESCRIPTION
**Pull request by natapg**
Closes #795
Mockup [figma.com/file/fTnEEueuxlkKrubDegp9aw/HP?node-id=775%3A41675&t=4d1tRbc6njRuAd62-0](https://www.figma.com/file/fTnEEueuxlkKrubDegp9aw/HP?node-id=775%3A41675&t=4d1tRbc6njRuAd62-0)
DP: [deploy-preview-827--fingerprintcom.netlify.app](https://deploy-preview-827--fingerprintcom.netlify.app/)

### CODE REVIEW

File: src/constants/env.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ export const BRANCH = process.env.BRANCH
 
 export const HOST =
   getContextEnv<string>({
-    [NetlifyContext.Production]: 'https://fingerprint.com',
+    [NetlifyContext.Production]: 'http://localhost:8000',
```
Comment (tan-90) [inline src/constants/env.ts:130]: Reminder to change back.

File: src/components/homepage/PlatformSection/PlatformSection.tsx [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
+            <DotSVG className={styles.dotLineIcon} />
```
Comment (tan-90) [inline src/components/homepage/PlatformSection/PlatformSection.tsx:64]: Since they all have the same class can't we use the `repeatElement` helper?

**Review by tan-90 — APPROVED** [review]

File: src/constants/env.ts (hunk 1 above)
Comment (natapg) [reply src/constants/env.ts:130]: oh I wasn't supposed to commit this file

File: src/components/homepage/PlatformSection/PlatformSection.tsx (hunk 2 above)
Comment (natapg) [reply src/components/homepage/PlatformSection/PlatformSection.tsx:64]: you're right :D

**Pull request merged**